workflow-types/src/lib.rs

```diff
@@ -59,12 +59,16 @@ impl Workflow {
         &self.shells
     }
 
-    pub fn new(name: impl Into<String>, command: impl Into<String>) -> Self {
+    pub fn new(
+        name: impl Into<String>,
+        command: impl Into<String>,
+        description: Option<String>,
+    ) -> Self {
         Workflow {
             name: name.into(),
             command: command.into(),
             tags: vec![],
-            description: None,
+            description,
```

Comment:
thoughts on setting this via a builder pattern instead? 
```rust
fn with_description(mut self, description: String) -> Self {
}
```

Passing this through the constructor can be a little awkward if majority of the time it will be just `None` (in which case it's not clear what the description even is). This also matches the API for setting the `arguments`

Reply:
Totally see that reasoning - will make the change!